### ISSUE TITLE
How to use my own dataset?

### ISSUE THREAD

**Issue by suayu** [swc-17/SparseDrive#46]
If I want to use my own nuscenes format data set for inference and training, what format does it need to meet and what content does it contain?

**Comment by swc-17** [comment]
Did not understand the question, please be more specific. If your dataset has the same format with nuScenes, it should be easy to transfer to your dataset.

**Comment by suayu** [comment]
What is the LIDAR data used for? There is no mention in the paper that the model uses LIDAR information. Can we replace LIDAR data with ego car coordinate truth values?

**Comment by swc-17** [comment]
Lidar data is used to generate depth GT for dense depth supervision in stage1. You can remove this and lower the learning rate to ensure the training stability.

**Comment by suayu** [comment]
Thanks.

**Comment by suayu** [comment]
请问如果我只提供六图环视图像数据，并只需要它根据这些视觉数据进行推理，不需要进行评估测试，该如何做？

**Comment by swc-17** [comment]
把config里eval_mode全部设为False。

**Comment by suayu** [comment]
那么我需要把我的六图环视图像数据放到哪个目录下呢？假设这些数据没有任何标注。

**Comment by swc-17** [comment]
需要更改nuscenes_infos_val.pkl中的图像路径为你自己的路径，还要更改对应的相机内外参、自车坐标变换等等。
